app/src/main/java/net/gsantner/opoc/util/Callback.java

```diff
@@ -37,6 +37,10 @@
         void callback(A arg1, B arg2, C arg3, D arg4, E arg5);
     }
 
+    public interface b0 {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The interface name 'b0' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5279643)